jekyll/_cci2/server-3-whats-new.adoc

```diff
@@ -25,7 +25,49 @@ toc::[]
 
 From the KOTS admin console dashboard, select Version History from the menu bar and click **Deploy** for server v3.2.0.
 
-When upgrading from server 3.1.x to 3.2 there will be some downtime due to a change to the PostgreSQL pod.
+When upgrading from server 3.1.x to 3.2 there will be some downtime due to a change to the PostgreSQL pod. There are two issues you could run into with this update, which are covered in the following sections.
+
+==== PostgreSQL pod stuck in `pending`
+If you find that the PostgreSQL pod is stuck in a `pending` state after upgrading, scale down the pods to 0 with the following command – this will scale and terminate the application pods without any data loss:
+
+To check if your postgreSQL pod is stuck in `pending`:
+
+```bash
+$ kubectl get pod -l app.kubernetes.io/name=postgresql
+NAME           READY   STATUS    RESTARTS   AGE
+postgresql-0   1/1     Pending   0          3m
```

Comment:
i think this needs one more ``` below this line

Reply:
Yes I'm just fixing this locally, like you say it doesn't work in suggest changes because it thinks the ``` is the end of the comment! I reckon that's a bug actually!